pkg/config/config_template.yaml

```
@@ -762,6 +762,16 @@ api_key:
   # obfuscation:
   #     <OBFUSCATION_CONFIGURATION>
 
+  ## @param filter_tags - object - optional
+  ## Defines rules by which to filter traces based on tags.
+  ##  * require - list of key or key/value strings - traces must have those tags in order to be sent to Datadog
+  ##  * reject - list of key or key/value strings - traces with these tags are dropped by the Agent
+  ## Note: Rules take into account the intersection of tags defined.
+  #
+  # filter_tags:
+  #     require: "<LIST_OF_KEY_VALUE_TAGS>"
+  #     reject: "<LIST_OF_KEY_VALUE_TAGS>"
```

I'm concerned that using quotes here might lead someone to believe that the value is a string, when in fact it's an array of strings. Perhaps this is better (for both):

```suggestion
  #     require: [<LIST_OF_KEY_VALUE_TAGS>]
  #     reject: [<LIST_OF_KEY_VALUE_TAGS>]
```